images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -331,6 +331,20 @@ function Get-DotNetCoreSdkVersions {
     return $dotNetCoreSdkVersion
 }
 
+function Get-DotnetTools {
+    $dotnetToolset = (Get-ToolsetContent).dotnet
+    $dotnetTools = $dotnetToolset.tools
+
+    $toolsList = @()
+
+    ForEach ($dotnetTool in $dotnetTools) {
+        $version = $dotnetTool.getversion
+        $toolsList += [PSCustomObject]@{
```

Comment:
Maybe we should return just an array of strings at line 343 without wrap to PsCustomObject?

Reply:
OK, but how should that be formatted properly so the caller can properly output the list? Can you please point me to an existing list that using this approach?

Reply:
@josesimoes something like this (I don't think we need this `v` before the version)?
```
foreach  ($dotnetTool in $dotnetTools) {
    $toolsList += $dotnetTool.name + " " + (Invoke-Expression $dotnetTool.getversion)
}
```
Though @al-cheb may have a better solution 😄